playwright/_impl/_selectors.py

```diff
@@ -46,6 +47,9 @@ async def register(
             await channel._channel.send("register", params)
         self._registrations.append(params)
 
+    def set_test_id_attribute(self, attribute_name: str) -> None:
+        set_test_id_attribute_name(attribute_name)
```

Comment:
```suggestion
        Locator.set_test_id_attribute_name(attribute_name)
```

and inside Locators class:

```python
    @staticmethod
    def set_test_id_attribute_name(date):
        ...
```

Reply:
I'll align from the other side!